### ISSUE TITLE
Update Business rules description

### ISSUE THREAD

**Issue by ishuverma** [PatrickSteiner/IoT_Summit_Lab#9]
In the business rules lab, the description of the rule trigger needs to be updated for values 0, 1, 2

**Comment by PatrickSteiner** [comment]
Not sure what you mean by this. Do you refer to image https://github.com/PatrickSteiner/IoT_Summit_Lab/blob/master/BusinessRulesService/images/allRules.png

**Comment by ishuverma** [comment]
yes, we need to update it with 2 rules so we can do all rules in lab 5. I'll update the instructions and image. you can close the issue.
